src/connection_string.ts

```diff
@@ -368,9 +370,11 @@ export function parseOptions(
     allKeys,
     Array.from(Object.keys(OPTIONS)).map(s => s.toLowerCase())
   );
+  const optionWord = unsupportedOptions.size > 1 ? 'options' : 'option';
+  const isOrAre = unsupportedOptions.size > 1 ? 'are' : 'is';
   if (unsupportedOptions.size !== 0) {
```

Comment:
```suggestion
  if (unsupportedOptions.size !== 0) {
    const optionWord = unsupportedOptions.size > 1 ? 'options' : 'option';
    const isOrAre = unsupportedOptions.size > 1 ? 'are' : 'is';
```
Just a suggestion to move these checks inside the if block that throws the error.

Reply:
That's actually where I was intending to put them, not sure how they ended up above it - good catch!